### PR TITLE
Alps 587 - Bad sync on director's cut

### PR DESCRIPTION
**Pull request by aberthet-gpsw**
- Fix bad timing synchronization in the Director and CameraAnimation.
- Took into account the onSeeked event.

### CODE REVIEW

**Comment by sw-team-release-gpsw** [comment]
[CLA assistant check](https://cla.gopro.com/sign) <br/>All committers have signed the CLA.

File: src/director/Director.js [hunk 1]
```diff
@@ -245,6 +246,11 @@ FORGE.Director.prototype._playHandler = function()
     if (this._track !== null)
     {
         this.play(this._track);
+
+        if (typeof this._viewer.story.scene.config.media !== "undefined" && this._viewer.story.scene.config.media.type === FORGE.MediaType.VIDEO)
```
Comment (rroux-gpsw) [inline src/director/Director.js]: FYI I think that you can just check `viewer.story.scene.media.type`

File: src/director/Director.js [hunk 2]
```diff
@@ -287,13 +294,19 @@ FORGE.Director.prototype._playingHandler = function()
...
```
Comment (rroux-gpsw) [inline src/director/Director.js]: FYI I think that you can just check `viewer.story.scene.media.type`

**Comment by bbaudel-gpsw** [comment]
Sometimes the bad sync is corrected but not in all cases. Tested with low bandwidth.

File: src/director/Director.js (hunk 1 above)
Comment (aberthet-gpsw) [inline src/director/Director.js]: Yep, changed all tests like this one in the file.

**Pull request merged**